functions/go/apply-setters/README.md

```diff
@@ -0,0 +1,94 @@
+# apply-setters
+
+### Overview
+
+<!--mdtogo:Short-->
+
+Apply setter values on resources fields. May set either the complete or partial field value.
+
+<!--mdtogo-->
+
+Setters provide a solution for template-free setting of field values. They are a
+safer alternative to other substitution techniques which do not have the context
+of the structured data. Setters may be invoked to modify the configuration
+using `apply-setters` function to set values.
+
+### Synopsis
+
+<!--mdtogo:Long-->
+
+```
+kpt fn eval apply-setters:VERSION [DIR] -- [setter_name=setter_value]
+```
+
+Data model
```

Comment:
I think you can remove lines 24 to 46, they're not adding much value.

Reply:
Done.